src/Select.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class Select extends Component {
+    render() {
+
+        return (
+            <div className="form-group">
+                <select name={this.props.name} type={this.props.type} className="select-list">
+                    <option value="volvo">Volvo</option>
```

Comment:
Same here with the options: This is not a custom component if you hardcode car brands.